src/adapters/net.cpp

```diff
@@ -204,6 +204,11 @@ namespace net {
     } else {
       m_tuntap = false;
     }
+
+    if (strncmp(driver.driver, "bridge", 3) == 0) {
```

Comment:
```suggestion
    if (strncmp(driver.driver, "bridge", 6) == 0) {
```

Otherwise this only compares the first three bytes